src/ccid_serial.c

```diff
@@ -833,6 +850,11 @@ status_t OpenSerialByName(unsigned int reader_index, char *dev_name)
 		unsigned char tx_buffer[] = { 0x02 };
 		unsigned char rx_buffer[50];
 		unsigned int rx_length = sizeof(rx_buffer);
+		
+		if (0 == strcasecmp(reader_name,"SEC1210"))
+		{
+			tx_buffer[0] = 0x06;
+		}
```

Comment:
What is this command 0x06?
The command 0x02 used to get the firmware does not work?